pyproject.toml

```diff
@@ -73,7 +73,9 @@ simpleitk = "^2.3.1"
 openpyxl = "^3.1.2"
 monai = "^1.3.0"
 nnunetv2 = "^2.3.1"
-picai_eval = "^1.4.6"
```

Comment:
Always nice to remove dependencies! Especially with this package its not that well maintained